x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -422,7 +424,9 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
           }
           const warning = validateFilePathInput({ os, value: wildcardValue });
           actualWarning =
-            warning === FILENAME_WILDCARD_WARNING ? getWildcardWarning(warning) : warning;
+            warning === FILENAME_WILDCARD_WARNING
+              ? warning && getWildcardWarning(warning)
```

Comment:
Was curious if `warning &&` was necessary.... [Working as intended...](https://github.com/microsoft/TypeScript/issues/39550#issuecomment-660534514) 😅

Reply:
Ah ok cool! I'll remove.

Reply:
Oh you can't remove it, haha 


<img width="1197" alt="image" src="https://user-images.githubusercontent.com/2946766/232613094-44bd03cf-14d6-4564-9ba9-9a344d03daef.png">